CustomBatteries/Items/CbCore.cs

```diff
@@ -186,6 +186,29 @@ protected void CreateIngredients(IEnumerable<TechType> parts, List<Ingredient> p
 
         protected abstract string[] StepsToFabricatorTab { get; }
 
+        private bool? _placeBatteriesFeatureEnabled = null;
+        protected bool PlaceBatteriesFeatureEnabled
```

Comment:
`_placeBatteriesFeatureEnabled ` and `PlaceBatteriesFeatureEnabled` can be made `static` since we on;y need to check this once.

👍